test-e2e/Cargo.toml

```diff
@@ -13,8 +13,12 @@ name = "ios"
 [[bin]]
 name = "android"
 
+[[bin]]
+name = "web"
+
 [dependencies]
 mopro-ffi = { path = "../mopro-ffi", features = ["halo2", "circom", "ashlang"] }
+mopro-wasm-lib = { path = "./mopro-wasm-lib" }
```

Comment:
here should be `mopro-wasm = { path = "../mopro-wasm" }`
because you need to use `mopro_wasm::app_config::web::build()`
(it should work just like `mopro-ffi`)

`mopro-wasm-lib` is used for wasm library / mostly for the nightly environment